src/Helpers/DocumentCreation/DocumentCreationHelper.cs

```diff
@@ -15,6 +15,9 @@ public async Task<List<string>> GenerateQuestionAndAnswersList(FormAnswers formA
             var summaryBuilder = new SummaryAnswerBuilder();
             var reducedAnswers = FormAnswersExtensions.GetReducedAnswers(formAnswers, formSchema);
 
+            if (formAnswers.CaseReference != string.Empty)
+                summaryBuilder.Add("Case Reference", formAnswers.CaseReference, Enum.EElementType.Textbox);
```

Comment:
Rather prefer the "!string.IsNullOrEmpty()" check, which we mostly if not solely use.